windows/src/desktop/history.md

```diff
@@ -0,0 +1,359 @@
+* Fixed: Crash in Keyman Configuration [kmshell.exe_9.0.451.0_script_TfrmMain_0] (I4251)
+* Fixed: kmtip install does not register Win 8 support features (I4252)
+* Fixed: TSF deadkeys do not function correctly (I4262)
+* Fixed: Test for text editor running fails (I4265)
+
+## 9.0.451 beta
+* Fixed: Keyman Configuration crashed on first run due to koKeymanUniscribeManager reference (I4250)
+
+## 9.0.450 beta
+* Fixed: If kmtip CKMTipTextService::Activate fails, cleanup (I3706)
+* Minor: Refactor kmxfile utility functions (I3757)
+* Added: Removed all legacy keyboard management Win32 API calls and use only TSF (I4220)
+* Fixed: Crash when OSK closed/reopened without dismissing hint window [CrashID:keyman.exe_9.0.449.0_2C405C5D_EInvalidPointer] (I4242)
+
+## 9.0.449 beta
+* Opening font helper or keyboard usage from Keyman menu on Win 8 still shows HTML outside window (I4225)
+* Excmagic.debug left scattered around program file directories after uninstall (I4218)
+
+## 9.0.448 beta
+* Shift + Arrows do not select text (only move caret) in Win 8 when Keyman keyboard is active (I4201)
+* Keyman TIP should use ITfTextInputProcessorEx (I4216)
+* Keyman leaks an Internet Explorer window handle (I4214)
+* Help dialog appears below OSK and is inaccessible (I4209)
+* Font helper and Keyboard usage appear outside frame in Win 8 (I4208)
+* Shift states still not working with unprocessed keys in V9 (I4128)
+* Activate/Purchase dialogs are incomplete and the Buy Modules button doesn't work (I4090)
+
+## 9.0.447 beta
+* Exit Keyman hint appears to be blank on Win8? (I4187)
+* Pressing Enter in install keyboard dialog gives error about admin req (I4172)
+* Help Contents link does not work from Keyman menu (I3993)
+* Help window Help and Help on Keyboard links don't work (I3676)
+* Base Keyboard dialog has wrong style of buttons (I4184)
+* Use TTempFileManager for all temporary files (I4195)
+* Lang switch window shows wrong selection with Alt+LeftShift when TIP is active (I4191)
+* keyman.exe seems to be missing icon (I3769)
+* Lang switch window shifts on first view (I4190)
+* wm_kmmoreposting must be refactored for TIP work as it is not sequential (I4196)
+* Avoid interactions with full-screen RDP (I4197)
+* mcompile logs should be stored in diag folder (I4174)
+* Uninstalling a keyboard leaves the mnemonic recompiled layouts behind (I4173)
+
+## 9.0.446 beta
+* Keyman Engine installer does not include mcompile.exe (I4171)
+
+## 9.0.445 beta
+* Mnemonic layouts should be recompiled to positional based on user-selected base keyboard (I4169)
+* Console execute in utilexecute.pas needs a temp copy of buffer to avoid write access violations (I4170)
+* Shift states still not working with unprocessed keys in V9 (I4128)
+    
+## 9.0.444 beta
+* Keyman Desktop installer does not install x64 TIP (I4161)
+
+## 9.0.442 beta
+* Add keyboard version information to Keyman Configuration (Tweak) (I4136)
+
+# Old Release Notes - What's New?
```

Comment:
In Developer, this section was 
```
# Legacy Release Notes - What's New?
```
Maybe we can pick one to be consistent

Reply:
That's fair.